engine/runner/pooled.go

```diff
@@ -126,6 +126,8 @@ func (runner *PooledRunner) RunAction(ctx context.Context, act action.Action, in
 		if logger.DebugEnabled() {
 			logger.Debugf("Action '%s' returned", support.GetRef(act))
 		}
+		// Release memory
```

Comment:
Not sure why this is necessary, that reference should be released when the function returns